scripts/main.js

```diff
@@ -72,9 +72,51 @@ const departments = {
+//     "4: [Marktonderzoeksanalist],\n")
+//
+// if (userInput === "0") {
+//     console.log("Je koos marketingmanager. Een uitdagende rol! De marketingmanager is verantwoordelijk voor het ontwikkelen en implementeren van marketingstrategieën om de producten of diensten van een bedrijf te promoten. Ze houden toezicht op marktonderzoek, reclamecampagnes en initiatieven voor klantenwerving. Ze analyseren markttrends, identificeren doelgroepen en werken samen met interdisciplinaire teams om marketingdoelen te bereiken en bedrijfsgroei te stimuleren.")
+// }
+// if (userInput === "1") {
+//     console.log("Je koos Digital Marketing Specialist. Een uitdagende rol! Een Digita Marketing Specialist richt zich op online marketingactiviteiten om dezichtbaarheid van het merk, websiteverkeer en klantbetrokkenheid te vergroten. Z maken gebruik van verschillende digitale kanalen zoals sociale media,e-mailmarketing, zoekmachineoptimalisatie (SEO) en betaalde advertenties (PPC).Ze monitoren de prestaties van campagnes, analyseren gegevens en optimaliserenstrategieën om online aanwezigheid te maximaliseren en leads of conversies tegenereren.")
+//     }
+// if (userInput === "2") {
+//     console.log("Je koos Contentmarketeer. Een uitdagende rol! Een contentmarketeer is verantwoordelijk voor het ontwikkelen en uitvoeren van contentstrategieën die aansluiten bij het merk en de marketingdoelstellingen van een bedrijf. Ze houden toezicht op contentcreatie, waaronder blogs, artikelen, video's, infographics en berichten op sociale media. Ze werken samen met schrijvers, ontwerpers en andere belanghebbenden om hoogwaardige content te produceren die de doelgroep aanspreekt en merkbekendheid, thought leadership en klantenwerving stimuleert.")
+// }
+// if (userInput === "3") {
+//     console.log("Je koos Branding Agent. Een uitdagende rol! Een Branding Agent is verantwoordelijk voor het vaststellen en handhaven van een consistente merkafbeelding en identiteit op verschillende marketingkanalen. Ze ontwikkelen richtlijnen voor het merk, positionering en berichtenstrategieën. Ze houden toezicht op productlanceringen, reclamecampagnes en inspanningen op het gebied van merkcommunicatie. Ze volgen de prestaties van het merk, voeren marktonderzoek uit en doen aanbevelingen om de merkperceptie en merkwaarde te verbeteren.")
+// }
+// if (userInput === "4") {
+//     console.log("Je koos Marktonderzoeksanalist. Een uitdagende rol! Een marktonderzoeksanalist voert diepgaand onderzoek uit om gegevens te verzamelen en te analyseren over markttrends, consumentenvoorkeuren en concurrentieactiviteiten. Ze ontwerpen en voeren enquêtes, focusgroepen en interviews uit om relevante informatie te verzamelen. Ze interpreteren gegevens, genereren inzichten en bereiden rapporten voor met bruikbare aanbevelingen. Ze helpen bedrijven geïnformeerde beslissingen te nemen over productontwikkeling, doelmarkten, prijsstrategieën en marketingcampagnes.")
+// }
+// if (userInput !== true) {
```

Comment:
In de console kun je zien dat nu je `if (userInput !== true)` altijd wordt uitgevoerd. Dat komt ten eerste omdat je geen `if (), else if (), else ()` statements gebruikt en dus alle if-statements doorlopen worden, ipv stoppen nadat er aan een coniditie is voldaan. En ten tweede omdat `userInput` altijd niet gelijk is aan 'true', aangezien het de waarde true of false niet kan aannemen.